### PR TITLE
Improving definitions for 'navigational' predicates. Fixes #523.

### LINKED ISSUE #523: Remove navigational parent and child predicates
gist 9.7.0 defines the following predicates (the latter two are for non-polyhierarchical trees):

```
hasSubCategory
hasNavigationalChild

hasSuperCategory
hasNavigationalParent

hasUniqueSuperCategory
hasUniqueNavigationalParent
```

I propose removing the navigation predicates,  as I don't see any use for them that can't be handled with the category predicates. Judging from the names, they may have been created for faceting, but they still seem redundant to me.

### PR DESCRIPTION
**Pull request by dylan-sa**

### CODE REVIEW

File: gistCore.ttl [hunk 1]
```diff
@@ -3260,13 +3260,15 @@ gist:hasMultiplier
 gist:hasNavigationalChild
 	a owl:ObjectProperty ;
 	owl:inverseOf gist:hasNavigationalParent ;
-	skos:definition "Used for informal hierarchical taxonomies.  Supports polyhierarchies"^^xsd:string ;
+	skos:definition "Used in cases where the child category is not necessarily an instance of the parent category but it is useful to represent the relationship hierarchically. Supports polyhierarchies."^^xsd:string ;
```
Comment (rjyounes) [inline gistCore.ttl:3263]: * A scope note could start with "Used in..." but a definition should not. 
* I don't see a need to mention poly-hierarchies here. 
* Can we make explicit mention of facets?

File: gistCore.ttl [hunk 2]
```diff
@@ -3260,13 +3260,15 @@ gist:hasMultiplier
...
+	skos:example "Refrigerator handles are not refrigerators, but it may be useful in some contexts (e.g., within a user interface) them to fall under the refrigerators category."^^xsd:string ;
```
Comment (rjyounes) [inline gistCore.ttl:3264]: * You're missing "for" before "them."
* Can we say something like "in faceted UI filters"?

File: gistCore.ttl [hunk 3]
```diff
@@ -3260,13 +3260,15 @@ gist:hasMultiplier
...
 	skos:prefLabel "has navigational child"^^xsd:string ;
 	.
 
 gist:hasNavigationalParent
 	a owl:ObjectProperty ;
-	skos:definition "Used for informal hierarchical taxonomies.  Supports polyhierarchies"^^xsd:string ;
+	skos:definition "Used in cases where the child category is not necessarily an instance of the parent category but it is useful to represent the relationship hierarchically. Supports polyhierarchies."^^xsd:string ;
+	skos:example "Refrigerator handles are not refrigerators, but it may be useful in some contexts (e.g., within a user interface) them to fall under the refrigerators category."^^xsd:string ;
```
Comment (rjyounes) [inline gistCore.ttl:3271]: Same as above.

File: gistCore.ttl [hunk 4]
```diff
@@ -3430,7 +3432,7 @@ gist:hasUniqueNavigationalParent
 		owl:FunctionalProperty
 		;
 	rdfs:subPropertyOf gist:hasNavigationalParent ;
-	skos:definition "Used for taxos that must have single parents"^^xsd:string ;
+	skos:definition "Used in cases where the child category has a unique parent that it is not necessarily an instance of but it is useful to represent the relationship hierarchically."^^xsd:string ;
```
Comment (rjyounes) [inline gistCore.ttl:3435]: Awkward wording. This needs to be smoothed out to be comprehensible.

**Review by rjyounes — CHANGES_REQUESTED** [review]
The wording of these annotations, and a better distinction between definitions and scope notes, needs to be thought through more carefully.

**Comment by dylan-sa** [comment]
@rjyounes All good points. I am pushing some changes to address. I have re-worked the `skos:definition`s so that they are now closer to proper distinctionary definitions. The definitions are now unique to each property and indicate directionality. (@wslooney gets credit for some of the improvement here.)

**Review by rjyounes — APPROVED** [review]

**Pull request merged**